lib/ovirt/vm.rb

```diff
@@ -80,6 +80,18 @@ def move(storage_domain)
       action['href']
     end
 
+    def boot_order=(devices)
+      devices = [ devices ].flatten
+      update! do |xml|
+        xml.os do
+          devices.each do |d|
+            xml.boot(:dev => d)
+          end
+        end
+        xml.memory self[:memory]  # HACK: RHEVM BUG: RHEVM resets it to 10GB, unless we set it
```

Comment:
Is this necessary?  I didn't see any problems in my testing against v3.3 or v3.4.  I don't remember if this problem was in updating specific attributes or if it was version dependent.